lesson01/assignments/Part2/scripts.js

```diff
@@ -0,0 +1,109 @@
+// jQuery Doc Ready
+$(function () {
```

Comment:
You don't need a $(function()) for each of your functions.  You only need one and then you can put all your code inside it.